ui/plugins/failure_summary/main.html

```diff
@@ -63,7 +63,20 @@
 
         </li>
 
-        <li ng-if="!tabs.failureSummary.is_loading && jobLogsAllParsed && bugSuggestionsLoaded && job_log_urls.length && suggestions.length == 0">
+        <span ng-if="errors.length > 0">
+          <span><b>Failed Buildbot Steps</b></span>
```

Comment:
Thinking further, perhaps better wording here would be, "Unsuccessful Execution Steps".  I know that's a bit wordy, but the lines aren't necessarily failures.  They're just "not all the way successful."  A nuanced difference, I know...  :)  
It would be nice, too, to have a line above this that says, "No Bug Suggestions Available".  So:

    No Bug Suggestions Available
    Unsuccessful Execution Steps:
        o warning, bad thing happened here.  View log

Something along those lines.  :)